models/db.go

```diff
@@ -0,0 +1,23 @@
+package models
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+)
+
+func OpenDbConnection() (*gorm.DB, error) {
+	newLogger := logger.New(log.New(os.Stdout, "\n", log.LstdFlags),
+		logger.Config{
+			SlowThreshold: time.Second,
+			LogLevel:      logger.Info,
+			Colorful:      true,
+		},
+	)
+
+	return gorm.Open(sqlite.Open("test.db"), &gorm.Config{Logger: newLogger})
+}
```

Comment:
I am not happy with the db.go in the models folder. However, this is not some complicated system so i'll allow it :-)

Reply:
> I am not happy with the db.go in the models folder. However, this is not some complicated system so i'll allow it :-)

I fully agree, but i feel like our root had gotten a bit overcrowded. I might be better to place it in repositories/db.go